doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of util.inherits() is discouraged. Please use the ES6 `class` and
+`extends` keywords to get language level inheritance support. Also note that
+the two styles are semantically incompatible._
```

Comment:
Maybe this is the wrong place, but a link to explanation of `semantically incompatible` would be helpful if there is a good one, and we are ok with external links. Perhaps even just a link in the PR so it can be found by tracing back to this PR.